contrail-api/functions/actions/resource.js

```diff
@@ -271,6 +271,9 @@ checkUserPermissionsForResource = async (userId, resourceId) => {
     }
 }
 
+const getFileFromBucket =  (ownerId, resourceName) => {
+    return bucket.file(`${ownerId}/${resourceName}`);
```

Comment:
this is not needed. its only one line

Reply:
What if we change the bucket path though!! :/